kubeflow/contrib/kserve/Makefile

```diff
@@ -10,7 +10,11 @@ KFCTXT=$(NAME)
 .PHONY: apply
 apply: hydrate
 # Apply App kserve
+	mkdir $(build_dir)/runtimes
+	mv $(build_dir)/serving*clusterservingruntime* $(build_dir)/runtimes/
```

Comment:
Add a comment here to explain why do we move the pattern `serving*clusterservingruntime*` to runtimes/ folder?

Reply:
Good point. Fixed. Thanks!